qemu/tests/cfg/virtio_mem_hugetlb_migration.cfg

```diff
@@ -0,0 +1,38 @@
+- virtio_mem_hugetlb_migration:
+    only Linux
+    no Host_RHEL.m7 Host_RHEL.m8 Host_RHEL.m9.u0 Host_RHEL.m9.u1 Host_RHEL.m9.u2 Host_RHEL.m9.u3
+    no RHEL.7 RHEL.8
+    no s390x
+    type = virtio_mem_hugetlb_migration
+    virt_test_type = qemu
+    smp = 8
+    vcpu_maxcpus = ${smp}
+    threshold = 0.025
+    slots_mem = 20
+    maxmem_mem = 80G
+    size_mem_vmem0 = 8G
+    mem = 8192
+    size_mem_mem0 = ${size_mem_vmem0}
```

Comment:
The relationship is incorrect although the value is correct:
`size_mem_mem0` should equal to the `mem`, 
The hugepage setting use `mem` , but it be used by the `vmem0`,  so as a workaround, the max `vmem0` is mem.

If the unit is a problem and do not want to make things complex, let's back to the initial design:
```
mem = 8192
size_mem_mem0 = 8G
# This is workaround to use the hugepages be created based on the size of mem.
size_mem_vmem0 = ${size_mem_mem0}  
```

Reply:
Updated !